app/controllers/application_controller.rb

```diff
@@ -210,6 +210,11 @@ def render_component(component_class, *args, **kwargs)
     render component_class.new(*args, **kwargs), content_type: "text/html"
   end
 
+  def creator_onboarding_mode
+    current_user.creator? && current_user.saw_onboarding == false
```

Comment:
Nitpick: `saw_onboarding` is a boolean column, they automatically get predicate methods.

```suggestion
    current_user.creator? && !current_user.saw_onboarding?
```

Reply:
Thanks for these @citizen428 ✨ This is just a rough MVP to see if the concept worked so I didn't pay too much attention as yet.